agent-ndk/src/main/java/com/newrelic/agent/android/ndk/ANRMonitor.kt

```diff
@@ -48,8 +48,12 @@ open class ANRMonitor {
                     runner.wait(ANR_TIMEOUT)
 
                     if (!runner.signaled) {
+                        AgentNDK.log.error("ANR monitor signaled false, ANR detected")
                         notify()
                         runner.wait()
```

Comment:
Why wait() here? That should cause the thread to block.

Reply:
This potion is part of the original code, we can remove it but what was the reason we had it before?